controllers/node.go

```diff
@@ -85,24 +86,18 @@ func (c *APIController) AddNode() {
 		panic(err)
 	}
 
-	if node.Id == "" || node.Name == "" || node.TabId == "" || node.PlaneId == "" {
+	if node.Name == "" || node.TabId == "" || node.PlaneId == "" {
 		resp = Response{Status: "fail", Msg: "Some information is missing"}
 		c.Data["json"] = resp
 		c.ServeJSON()
 		return
 	}
-
-	if object.HasNode(node.Id) {
-		resp = Response{Status: "fail", Msg: "Node ID existed"}
-		c.Data["json"] = resp
-		c.ServeJSON()
-		return
-	}
-
+	nodeNum := object.GetNodesNum() + 1
+	nodeId = node.ParentNode + node.Name + strconv.Itoa(nodeNum)
+	node.Id = nodeId
 	node.CreatedTime = util.GetCurrentTime()
 	res := object.AddNode(&node)
-	resp = Response{Status: "ok", Msg: "success", Data: res}
```

Comment:
- If the value of `res` is `false`, then `Status` should not be `"ok"`
- `Msg` should be the reason for the failure when adding node failed
- `Data` can be empty because the front end do not use it

Reply:
But in other methods, it also returns false directly.Do I need to judge whether res is false on this method alone？
![image](https://user-images.githubusercontent.com/55344330/120101725-0345a980-c17a-11eb-8ad6-86a3ec8b4b5c.png)

Reply:
In this project, all `object.*` functions related to DB will panic if there have any errors. 
And the implement of xorm is converting into a SQL statement similar to `Insert into table values (),(),()`.
So this `affected` may always be the num of your inserts(please correct me if there are any mistakes).
Above, I think there is no need to judge again.

Reply:
Thank you for your guidance